src/ui-kit/SliderYears/index.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+import { getEndYear, getStartYear } from 'redux/selectors'
+import { setCurrentYearsDate, setCurrentYearsAfter } from 'redux/actions'
+
+import { number } from 'prop-types'
+
+import { Slider } from 'antd'
+
+import './sliderYears.scss'
+
+
+const SliderYears = ({ setCurrentYearsDate, setCurrentYearsAfter, startYear, endYear }) => {
+
+  const currentDate = new Date()
+  const currentYear = currentDate.getFullYear()
```

Comment:
you are doing this every time on every render - it's very bad.
Can you find a solution to not do it on each render ?